README.md

```diff
@@ -1,36 +1,31 @@
-[![Manager](https://github.com/thoughtworks/maeve-csms/workflows/Manager/badge.svg)](https://github.com/thoughtworks/maeve-csms/actions/workflows/manager.yml)
-[![Gateway](https://github.com/thoughtworks/maeve-csms/workflows/Gateway/badge.svg)](https://github.com/thoughtworks/maeve-csms/actions/workflows/gateway.yml)
+[![Manager](https://github.com/thoughtworks/Maeve-csms/workflows/Manager/badge.svg)](https://github.com/thoughtworks/Maeve-csms/actions/workflows/manager.yml)
+[![Gateway](https://github.com/thoughtworks/Maeve-csms/workflows/Gateway/badge.svg)](https://github.com/thoughtworks/Maeve-csms/actions/workflows/gateway.yml)
 
 # MaEVe
 
-MaEVe is an EV charge station management system (CSMS). It began life as a simple proof of concept for
-implementing ISO-15118-2 Plug and Charge (PnC) functionality and remains a work in progress. It is hoped that over
-time it will become more complete, but already provides a useful basis for experimentation.
+Maeve is an EV Charge Station Management System (CSMS). It began life as a simple proof of concept for
```

Comment:
The change in capitalization of MaEVe now makes it inconsistent with the title and MaEVe was intended to highlight the EV.

Reply:
It's ok to highlight _EV_ in the header, but not every time you write it. Too easy to make mistakes.

Reply:
My 2 cents: afaik there's no official way to write MaEVe. Personally I tend to capitalise the EV when I'm writing to an audience that may be new to EV or Maeve. I like the ability for the author to write as preferred, analogous to how I like that its meaning can be moulded.

Reply:
My English teacher always said to me mid-caps in written English is bad. 😆